ui/.prettierrc

```diff
@@ -1,17 +1,16 @@
 {
-  "semi": false,
-  "trailingComma": "none",
-  "printWidth": 120,
-  "singleQuote": true,
-  "useTabs": false,
-  "tabWidth": 2,
   "arrowParens": "always",
-  "jsxSingleQuote": false,
-  "jsxBracketSameLine": false,
   "bracketSpacing": true,
-  "parser": "typescript",
```

Comment:
"parser": "typescript" is removed since it seems to interfere with VSC Document Format function.
"printWidth": 40 instead of 100 to force attributes on each line.
The rest is just sorted.